src/tyr/serializers.cc

```diff
@@ -157,6 +158,18 @@ void openlr(const valhalla::Api& api, int route_index, rapidjson::writer_wrapper
   writer.end_array();
 }
 
+void serializeWarnings(const valhalla::Api& api, rapidjson::writer_wrapper_t& writer){
+    valhalla::Info warning;
```

Comment:
does this work for you?.. don't you have to use the `api` above to see the warning which were added during processing?

Reply:
I've refactored the function to use the `api` directly.